### DISCUSSION TITLE
dont backuping game in disk, only game memory

### DISCUSSION THREAD

**Discussion by gotdemon1** [wolveix/satisfactory-server#305] [General]
Today, when I restarted the game, I found that all the game's saves were only in the server's memory, and the disk was empty. The backup folders are empty and there is no save, I lost all progress. The rights to folders 777, files do not appear, what should I do?

**Comment by gotdemon1** [comment]
![2](https://github.com/user-attachments/assets/25fa1cdf-c3fd-4f60-b896-3620a9671565)
![1](https://github.com/user-attachments/assets/1fb69b07-18b1-430d-917e-82965d86eef9)

**Comment by wolveix** [comment]
Really sorry about that, please check #266

This was a bug when 1.0 first dropped, and affected anyone who pulled the image within the first hour or so of the workaround being published. Please update to the latest image.

**Comment by gotdemon1** [comment]
you are the great, thank you
